ddl/ddl_api.go

```diff
@@ -202,12 +202,13 @@ func setCharsetCollationFlenDecimal(tp *types.FieldType) error {
 			}
 		}
 	}
-	// If flen is not assigned, assigned it by type.
+	// If flen or decimal is not assigned, assigned it by type.
```

Comment:
maybe this comment is more explicit: "use default value for Flen and Decimal when they are unspecified"